### ISSUE TITLE
Put up posters advertising our group in the TIB & LUH cafeterias

### ISSUE THREAD

**Issue by katrinleinweber** [TIBHannover/FAIR-studyGroup#21]
From [TeachTogether.tech/en/marketing](http://teachtogether.tech/en/marketing/#s:marketing-branding): **Unless** the room will already be full in October, we should get a few posters or hand-outs ready for **early 2019** to advertise:

- [x] [What we do](https://tibhannover.github.io/FAIR-studyGroup/#services)
- [x] URL with QR code to [our events portfolio](https://tibhannover.github.io/FAIR-studyGroup/#portfolio)
- [x] QR to event-suggestion instructions ~~Better to have only one, because the "Suggest..." link is on the homepage~~
- [x] fetch prints & pin up around TIB
- [x] print 2 small extras for directions within the Grunwald-Haus

Prerequisites:

- [x] find poster template of the [workshop](https://tibhannover.github.io/2018-07-09-FAIR-Data-and-Software/) & reuse it

**Comment by rbirkelbach** [comment]
I can send out some infos to the researchers and software developers at DZHW.

**Comment by katrinleinweber** [comment]
How about this wording:
![grafik](https://user-images.githubusercontent.com/9948149/46203985-e21ec200-c31b-11e8-90e9-64133e8bd7a9.png)

@rbirkelbach: Should/Can we add the DZHW logo?

**Comment by KBurkhard** [comment]
How about "Weekly Wednesdays" or just "Weekly") instead of "Every week"? And add "Check the events calendar for topics and submit your suggestions at: " and then come the QR and the link?

Maybe also add the address of the building or should they go to the website for that?

**Comment by katrinleinweber** [comment]
OK, done. I'm a bit worried that it will overloaded, but at least we'll now not put the burden on explaining where the event is on the info desk colleagues ;-)

Unfortunately, the design will be implemented externally, so I can't upload the draft here :-/

**Comment by katrinleinweber** [comment]
Done. Pinned up in the main building, with 2 more copies available.

**Comment by rbirkelbach** [comment]
Sorry, I didn't see your question! No, no need to put the dzhw logo because I join in my free time, not as somebody representing the dzhw.
